framework/tlvf/CMakeLists.txt

```diff
@@ -13,7 +13,8 @@ find_package(elpp REQUIRED)
 find_package(PythonInterp REQUIRED)
 
 set(TLVF_DIR ${CMAKE_CURRENT_SOURCE_DIR})
-set(PythonTlvf ${TLVF_DIR}/tlvf.py)
+set(PythonTlvf ${TLVF_DIR}/tlvf.py
+    CACHE INTERNAL "Path to tlvf script during build")
```

Comment:
The below install(PROGRAMS ${TLVF_DIR}/tlvf.py DESTINATION host/bin) should also be removed as part of this commit, and same for the target exporting (this goes for all target exports which are only used internally)